package.json

```diff
@@ -5,28 +5,31 @@
   "dependencies": {
     "@aragon/os": "3.1.9",
     "@aragon/client": "^1.0.0-beta.8",
-    "@aragon/ui": "^0.11.0",
+    "@aragon/ui": "^0.16.0",
     "react": "^16.3.2",
     "react-dom": "^16.3.2",
     "styled-components": "^3.2.6"
   },
   "devDependencies": {
     "@aragon/cli": "^4.0.6",
+    "babel-core": "^6.26.3",
     "babel-plugin-styled-components": "^1.5.1",
     "babel-plugin-transform-runtime": "^6.23.0",
     "babel-preset-env": "^1.6.1",
     "babel-preset-stage-2": "^6.24.1",
-    "parcel-bundler": "^1.9.3",
-    "truffle": "4.0.5"
+    "parcel-bundler": "^1.9.3"
   },
   "scripts": {
-    "start": "aragon run",
+    "start": "npm run start:app -- --open=\"\" & npm run start:aragon:http",
```

Comment:
I'm actually not sure about this. It was my best shot at attempting to both start the server and do `aragon run` so devs only have to execute one command if they want to.

The UX with this is that the output of both processes kind of merges, I thought about silencing parcel's output, but that could be problematic if there are errors or useful information printed.

There is also a race condition here that could be triggered if `aragon run` somehow finishes before parcel starts the server, which could cause a problem because the artifacts wouldn't be available when the CLI tries to fetch them to verify the publishing went well.

Reply:
I agree we shouldn’t do that at this level, but something we could do would be to use the [API of Parcel](https://parceljs.org/api.html), either directly inside aragon-cli, or from a dedicated script that would also import aragon-cli modules. That way, we could intercept messages and display them in a clean way.